### ISSUE TITLE
Remove line breaks from pasted DKIM pubkeys

### ISSUE THREAD

**Issue by pacien** [ovh-ux/ovh-manager-web#111]
### Description

It would be handy if line breaks were automatically removed when pasting a base64 DKIM key.

### Steps to Reproduce

1. Go to Domains -> DNS zone -> Add an entry -> DKIM
2. Fill the "Public key (base64)" field by pasting a PEM encoded key generated by openssl for example, without the header and footer.

**Expected behavior:**
Line breaks should be removed.

**Actual behavior:**
Each line break has to be removed by hand :(

**Reproduces how often:**
Everytime I mess up with my mail server configuration, i.e. quite often.

### Additional Information

None.

**Comment by marcozeus** [comment]
I don't know if it's the same issue but it could be very helpfull if DKIM entries were compatible with cPanel base64 DKIM string. Thank you very much.

**Comment by FredericEspiau** [comment]
Hey, thanks for your input !

@pacien do you know if this is still an issue ?

@marcozeus could you please give use some examples or where are the specs for cPanel base64 DKIM string ?

Thanks for helping us improve our products :)

**Comment by pacien** [comment]
@FredericEspiau Yes, the UI still rejects public keys with line breaks instead of stripping those automatically.

Rejected input:
```
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAn/8y3uYCQxSXZ58OYceG
A4uPdGHZXDYOQR11xcHTrH13jJEzdkYZG8irtyG+m3Jb6f9F8WkmTZxl+4YtkJdN
9WyrKhxq4Vbt42BthadX3Ty/pKkJ81Qn8KjxWoL+SMaCGFzRlfWsFju9Q5C7+aTj
eEKyFujH5bUTGX87nULRfg67tmtxBlT8WWWtFe2O/wedBTGGQxXMpwh4ObjLl3Qh
bfwxlBbh2N4471TyrErv04lbNecGaQqYxGrY8Ot3l2V2fXCzghAQg26Hc4dR2wyA
PPgWq78db+gU3QsePeo2Ki5sonkcyQQQlCkL35Asbv8khvk90gist4kijPnVBCuv
cwIDAQAB
```

Accepted input:
```
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAn/8y3uYCQxSXZ58OYceGA4uPdGHZXDYOQR11xcHTrH13jJEzdkYZG8irtyG+m3Jb6f9F8WkmTZxl+4YtkJdN9WyrKhxq4Vbt42BthadX3Ty/pKkJ81Qn8KjxWoL+SMaCGFzRlfWsFju9Q5C7+aTjeEKyFujH5bUTGX87nULRfg67tmtxBlT8WWWtFe2O/wedBTGGQxXMpwh4ObjLl3QhbfwxlBbh2N4471TyrErv04lbNecGaQqYxGrY8Ot3l2V2fXCzghAQg26Hc4dR2wyAPPgWq78db+gU3QsePeo2Ki5sonkcyQQQlCkL35Asbv8khvk90gist4kijPnVBCuvcwIDAQAB
```

**Comment by FredericEspiau** [comment]
@pacien 

Thanks for your input.

I've created an internal reference to make sure someone takes care of this: MANAGER-2119

**Comment by marcozeus** [comment]
Gulp! Now it works fine!
I'm sure that until one month ago this key:
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAsKzbfLHM5CF0S2jh98kHNTLt7fI/iw+6xSLfT+bxpwUrPDO5cTVYmzSBwtQZ5pr1c0q6o18Ccf4LWtrRpNifk0VcQiSupOCaHkL1rB7ZbxvgRt638vZqW+j3aBBOwWwXxyxfy+AAb/eIBzypBELFFGRabAVz5lxDlA/yOg61ieWkim7Ykdh1I83iLK1yW2+yByAoQ0Gu/9K32BsEypVZWB61f4Sd/wnZxIf6ETwohMqwjilK1gK2n0rXrj5X8mB/KKQjp9jyRmnH7O3jr/cyWxhfdjUHdyKwwixLD2xOA8nslRhsol7VOEo5rsyXFo0w8pNiYjqeqhtSWfY5Gi5fBwIDAQAB
didn't work. Infact your and Cpanel support teams told me to split the key into 2 txt strings.
Wow!
Anyway, thank you :-)

**Comment by FredericEspiau** [comment]
Well glad you're happy :)

**Comment by FredericEspiau** [comment]
@pacien your issue should be fixed by #892

**Comment by pacien** [comment]
@FredericEspiau I confirm. Thanks!
